pkg/operator/clustermemberremovalcontroller/clustermemberremovalcontroller_test.go

```diff
@@ -655,8 +657,11 @@ func TestAttemptToScaleDown(t *testing.T) {
 						m5Found = true
 					}
 				}
-				if m4Found && m5Found {
-					t.Fatal("expected either m4 or m5 to be deleted but not both")
+				if m4Found {
+					t.Fatal("expected m4 to be deleted")
+				}
+				if !m5Found {
+					t.Fatal("expected m5 not to be deleted")
```

Comment:
the way you wrote it, you can just error in the loop:

```
for _, member := range memberList {
   if member.ID == 4 {
       t.Errorf("not expected member with id %d", member.ID)
   }
}
```

or so, m5 not being deleted would've been caught by the member list length earlier already.